packages/sampler/stories/qa/ExpandableList.js

```diff
@@ -162,4 +162,16 @@ storiesOf('ExpandableList', module)
 		() => (
 			<ExpandableListWithAddedChildren />
 		)
+	)
+	.add(
```

Comment:
`ExpandableList` doesn't produce this bug since it is wrapped by `Toggleable` which provides the handlers. As a result, I'm going to remove the sample and rely on the unit tests to address the concern.